src/QSymbolicsBase/predefined_fock.jl

```diff
@@ -26,6 +26,21 @@ end
 SqueezedState(z::Number) = SqueezedState(z, inf_fock_basis)
 symbollabel(x::SqueezedState) = "0,$(x.z)"
 
+"""Two-mode squeezed vacuum state, or EPR state, in defined Fock basis."""
+@withmetadata struct EPRState <: SpecialKet
+    z::Number
+    basis::CompositeBasis
+    function EPRState(z::Number, basis::CompositeBasis)
+        bases = basis.bases
+        length(bases) == 2 && all(x -> x isa FockBasis, bases) || 
+                throw(ArgumentError(lazy"The underlying basis for an EPR state must be a
+                                        tensor product of two bases for single-mode quantum systems."))
```

Comment:
This might need to be a single line. Otherwise it prints a bit weird

![image](https://github.com/user-attachments/assets/ee7858b3-3075-443a-86d5-15e964021ca6)